src/initialization/MOM_state_initialization.F90

```diff
@@ -1416,7 +1422,8 @@ subroutine cut_off_column_top(nk, tv, GV, US, G_earth, depth, min_thickness, T,
     do k=1,nk
       call find_depth_of_pressure_in_cell(T_t(k), T_b(k), S_t(k), S_b(k), e(K), e(K+1), &
                                           P_t, p_surf, GV%Rho0, G_earth, tv%eqn_of_state, &
-                                          US, P_b, z_out, z_tol=z_tol)
```

Comment:
In MOM6, we find it very convenient to not put spaces around the `=` to help search for argument assignment as something separate from variable assignment, as described at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide#white-space.  Please change line 1426 to read `frac_dp_bugfix=frac_dp_bugfix)`, and similarly on lines 3150 and 3170.